sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -88,6 +89,9 @@ def update_table_mux_status_for_response_tbl(table_name, status, logical_port_na
     fvs = swsscommon.FieldValuePairs([('response', status)])
     table_name.set(logical_port_name, fvs)
 
+    if XCVRD_DEBUG_LOGS == True:
+        helper_logger.log_notice("Y_CABLE_DEBUG: Successful in returning probe port status {}".format(logical_port_name))
```

Comment:
Why not simply call `helper_logger.log_debug()`, and if we want to get these logs into the syslog, we can simply adjust the minimum log level to DEBUG when instantiating helper_logger?

Reply:
fixed

Reply:
Thanks for clarifying, I used syslog api's to change default log level through message now.